File: Client/ies/src/app/models/question.ts
import { Answer } from "./answer";

export class Question {
    questionId: number;
    description: string;
    possibleAnswers: Answer[]
    correctAnswerId: number;
    points: number;

    constructor() {
        this.questionId = 0
        this.description = ""
        this.possibleAnswers = []
        this.correctAnswerId = 0
        this.points = 0
    }
}

File: Server/InteractiveEducationalSoftware/InteractiveEducationalSoftware/IES.Models/DataModels/Question.cs
﻿using Dapper.Contrib.Extensions;

namespace IES.Models.DataModels
{
    public class Question
    {
        [Key]
        public int QuestionId { get; set; }
        public int ChapterId { get; set; }
        public string Description { get; set; }
        public List<Answer> PossibleAnswers { get; set; }
        public int CorrectAnswerId { get; set; }
        public int Points { get; set; }
    }
}
